docs/security/_toc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This folder will need an `index` page that the footer goes to. Consider using an index page like our migration guides https://quantum.cloud.ibm.com/docs/en/migration-guides